src/lib/types/TransferServiceData.ts

```diff
@@ -18,8 +20,9 @@ export type TransferResult = {
         | 'AasRegistry'
         | 'SubmodelRepository'
         | 'SubmodelRegistry'
-        | 'File transfer';
+        | 'FileTransfer';
     success: boolean;
+    // status: 'Success' | 'Failed' | 'Ignored' | 'Skipped';
```

Comment:
This came from the idea, that we may want to react differently based on the use case:
- If we did not push to a repository, we may want to ignore registry pushes for that aas/submodel
- If we push a second time after something did not work, we may want to skip already pushed files

This would give us the possibility to color code the results in a nice and easy way in the ui instead of toasters and working with the base error messages